diff --git a/node_modules/codemirror/addon/hint/javascript-hint.js b/node_modules/codemirror/addon/hint/javascript-hint.js
index 9f06b1b..56768c7 100644
--- a/node_modules/codemirror/addon/hint/javascript-hint.js
+++ b/node_modules/codemirror/addon/hint/javascript-hint.js
@@ -95,7 +95,7 @@
                     "lastIndexOf every some filter forEach map reduce reduceRight ").split(" ");
   var funcProps = "prototype apply call bind".split(" ");
   var javascriptKeywords = ("break case catch class const continue debugger default delete do else export extends false finally for function " +
-                  "if in import instanceof new null return super switch this throw true try typeof var void while with yield").split(" ");
+                  "if in import instanceof let new null return super switch this throw true try typeof var void while with yield").split(" ");
   var coffeescriptKeywords = ("and break catch class continue delete do else extends false finally for " +
                   "if in instanceof isnt new no not null of off on or return switch then throw true try typeof until void while with yes").split(" ");
 
